modules/ROOT/pages/upgrade-message-flows.adoc

```diff
@@ -1,8 +1,8 @@
 = Template Version Upgrades
```

Comment:
What templates? When? Why?
Consider something more specific and transitional from previous texts? I don't know if this is accurate, but something like "Improved B2B Transactions Through Runtime Template Upgrades"?